account_invoice_export/models/account_move.py

```diff
@@ -53,7 +53,6 @@ def _job_export_invoice(self, resend_invoice=False):
                     # The chatter of the invoice need to be updated, when the job fails
                     self.with_env(new_env).log_error_sending_invoice(values)
             raise
-        self.invoice_send = True
```

Comment:
time to test this? :)

Reply:
I do think to add a test for it would be good, yes